### DISCUSSION TITLE
Status "403" on startup

### DISCUSSION THREAD

**Discussion by derMiepz** [LOENS2/pterodactyl_exporter#37] [Q&A] [answered]
Heya!
So I just came across your Pterodactyl exporter and tried to set it up via an Egg, but I get an error 403.

- I created an API key, first with all read only, then all read/write permissions without success. 
- The panel is reachable from the outside with no firewall rules.
- The whole Pterodactyl setup (without your exporter) hosts a hand full of different servers for over two years now, so I'd say everything should be configured right.
- SSL/HTTPS is set up and working.
- When visiting `https://PANEL.DOMAIN.TLD/api/client/?type=owner&page=1` manually in the browser, I get a huge output with all the servers etc. 
- I tried changing the "Server List Type" from `owner` to `admin-all` with no success. 
- I tried adding `https://` before the hostname in the server configuration with no success. 
- Changing the `HTTPS` and `Ignore SSL` values from `0` to `1` didn't work. 
- Changing the Hostname from a domain to an IP didn't work. 

Now I'm slowly but surly running out of ideas and I hope you can help me to fix this issue :)
Best regards
~ Fabi
 
![image](https://github.com/user-attachments/assets/17f8123c-434a-461b-a1cb-e71caf41ce37)
![image](https://github.com/user-attachments/assets/c21c0c35-ee70-4c0c-b449-0b12f2b37225)

**Comment by LOENS2** [comment, marked as answer]
Your problem is that you are using a application API key rather than a client API key as the details key of the output of the exporter states. You can get the client API key under `Account Settings(the icon at the top, left of the logout button, on the servers screen) > API Credentials`. It should start with "ptlc_"

**Comment by derMiepz** [comment]
Ayy got it working! As for today, I didn't even know that there's two different APIs for Pterodactyl :D

Thank you very much for your quick response and have a nice weekend!
